packages/admin/admin/src/table/filterbar/filterBarPopoverFilter/FilterBarPopoverFilter.tsx

```diff
@@ -9,30 +10,132 @@ import { messages } from "../../../messages";
 import { dirtyFieldsCount } from "../dirtyFieldsCount";
 import { FilterBarActiveFilterBadgeProps } from "../filterBarActiveFilterBadge/FilterBarActiveFilterBadge";
 import { FilterBarButton, FilterBarButtonProps } from "../filterBarButton/FilterBarButton";
-import { FilterBarPopoverFilterClassKey, styles } from "./FilterBarPopoverFilter.styles";
 
 /**
  * @deprecated Use MUI X Data Grid in combination with `useDataGridRemote` instead.
  */
-export interface FilterBarPopoverFilterProps {
+
+export type FilterBarPopoverFilterClassKey =
+    | "root"
+    | "fieldBarWrapper"
+    | "popoverContentContainer"
+    | "paper"
+    | "buttonsContainer"
+    | "contentContainer";
+
+const Root = styled("div", {
+    name: "CometAdminFilterBarPopoverFilter",
+    slot: "root",
+    overridesResolver(_, styles) {
+        return [styles.root];
+    },
+})(
+    ({ theme }) => css`
+        background-color: ${theme.palette.common.white};
+        position: relative;
+        margin-bottom: 10px;
+        border-radius: 2px;
+        margin-right: 6px;
+    `,
+);
+
+const FieldBarWrapper = styled("div", {
+    name: "CometAdminFilterBarPopoverFilter",
+    slot: "fieldBarWrapper",
+    overridesResolver(_, styles) {
+        return [styles.fieldBarWrapper];
+    },
+})(
+    css`
+        position: relative;
+    `,
+);
+
+const PopoverContentContainer = styled("div", {
+    name: "CometAdminFilterBarPopoverFilter",
+    slot: "popoverContentContainer",
+    overridesResolver(_, styles) {
+        return [styles.popoverContentContainer];
+    },
+})(
+    css`
+        min-width: 300px;
+    `,
+);
+
+const StyledPopover = styled(Popover, {
+    name: "CometAdminFilterBarPopoverFilter",
+    slot: "paper",
+    overridesResolver(_, styles) {
+        return [styles.tabs];
+    },
+})(css`
+    margin-left: -1px;
+    margin-top: 2px;
+`);
+
+const ContentContainer = styled("div", {
+    name: "CometAdminFilterBarPopoverFilter",
+    slot: "contentContainer",
+    overridesResolver(_, styles) {
+        return [styles.contentContainer];
+    },
+})(
+    css`
+        box-sizing: border-box;
+        padding: 20px;
+        margin-bottom: 0;
+    `,
+);
+
+const ButtonsContainer = styled("div", {
+    name: "CometAdminFilterBarPopoverFilter",
+    slot: "buttonsContainer",
+    overridesResolver(_, styles) {
+        return [styles.buttonsContainer];
+    },
+})(
+    ({ theme }) => css`
+        border-top: 1px solid ${theme.palette.grey[100]};
+        justify-content: space-between;
+        box-sizing: border-box;
+        padding: 10px 15px;
+        display: flex;
+        height: 60px;
+    `,
+);
+
+export interface FilterBarPopoverFilterProps
+    extends ThemedComponentBaseProps<{
+        root: "div";
+        fieldBarWrapper: "div";
+        popoverContentContainer: "div";
+        paper: typeof Popover;
```

Comment:
`paper` is not correct here, this should probably be `popover`.
Previously the `paper` class was applied to the `Paper` inside the `Popover`, not the `Popover` itself. 

You can apply paper styles from the previous styling-method with the `sx` prop to the popover's paper.   

I don't think it's possible to apply the theme-styles of paper though... 
We probably need to remove the `paper` class and add a changelog for the breaking change, mentioning that the class key was removed and that this now needs to be done over the props (`slotProps` and `sx`).